http-api/src/issues.rs

```diff
@@ -53,7 +53,16 @@ async fn issues_handler(ctx: Context, project: Urn) -> Result<impl Reply, Reject
         .all(&project)
         .map_err(Error::Issues)?
         .into_iter()
-        .map(|(id, issue)| Cob::new(id, issue))
+        .map(|(id, mut issue)| {
+            if let Err(e) = issue
+                .resolve(storage.as_ref())
+                .map_err(Error::IdentityResolveError)
+            {
+                tracing::debug!("Failed to resolve issue author: {}", e);
```

Comment:
```suggestion
                tracing::error!("Failed to resolve issue author: {}", e);
```

Reply:
Would be good for this to stick out in the logs, or maybe `warn` is enough.

Reply:
Also might be handy to know the issue id in the log.

Reply:
Great, yeah that makes sense.